demos/serverless/src/handlers.js

```diff
@@ -272,3 +274,27 @@ function response(statusCode, contentType, body) {
     isBase64Encoded: false
   };
 }
+
+function addSignalMetricsToCloudWatch(logMsg, meetingId, attendeeId) {
+  const logMsgJson = JSON.parse(logMsg);
+  const metricList = ['signalingOpenDurationMs'];
+  for (let metricIndex = 0; metricIndex <= metricList.length; metricIndex += 1) {
+    const metricName = metricList[metricIndex];
+    if (logMsgJson.attributes.hasOwnProperty(metricName)) {
+      const metricValue = logMsgJson.attributes[metricName];
+      console.log('Logging metric -> ', metricName, ': ', metricValue );
+      const putMetric =
+        metricScope(metrics => async (metricName, metricValue, meetingId, attendeeId) => {
```

Comment:
- nit: `async` is not required since you haven't used any `await` keyword. Do you need to `await` on `putDimensions` and `putMetric` calls?
- Create `putMetric` outside the `for` loop. You don't need to initialize this function in every `for` loop iteration.

Reply:
We are using the for loop to iterate over the metrics in the metricList. We just have one metric in the list for now.
But as the list keeps growing, the for loop will call the metrics.putMetric() with the corresponding metricName and metricValue.